Les sous-répertoires linux-2.4.20/drivers/media/video.bt7/bttv9 et linux-2.4.20/drivers/media/video/bttv9 sont identiques.
diff -uN linux-2.4.20/drivers/media/video.bt7/Config.in linux-2.4.20/drivers/media/video/Config.in
--- linux-2.4.20/drivers/media/video.bt7/Config.in	2003-04-16 04:47:57.000000000 +0200
+++ linux-2.4.20/drivers/media/video/Config.in	2003-04-16 05:01:23.000000000 +0200
@@ -8,7 +8,8 @@
 dep_tristate '  I2C on parallel port' CONFIG_I2C_PARPORT $CONFIG_PARPORT $CONFIG_I2C
 
 comment 'Video Adapters'
-dep_tristate '  BT848 Video For Linux' CONFIG_VIDEO_BT848 $CONFIG_VIDEO_DEV $CONFIG_PCI $CONFIG_I2C_ALGOBIT $CONFIG_SOUND
+dep_tristate '  BT848 Video For Linux v7' CONFIG_VIDEO_BT848 $CONFIG_VIDEO_DEV $CONFIG_PCI $CONFIG_I2C_ALGOBIT $CONFIG_SOUND
+dep_tristate '  BT848 Video For Linux v9' CONFIG_VIDEO_BT848_9 $CONFIG_VIDEO_DEV $CONFIG_PCI $CONFIG_I2C_ALGOBIT $CONFIG_SOUND
 dep_tristate '  Mediavision Pro Movie Studio Video For Linux' CONFIG_VIDEO_PMS $CONFIG_VIDEO_DEV
 if [ "$CONFIG_ALL_PPC" = "y" ]; then
    dep_tristate '  PlanB Video-In on PowerMac' CONFIG_VIDEO_PLANB $CONFIG_VIDEO_DEV
diff -uNp t1/drivers/media/video/Makefile.dv05.orig t1/drivers/media/video/Makefile
--- t1/drivers/media/video/Makefile.dv05.orig	2003-08-01 17:06:06.000000000 +0200
+++ t1/drivers/media/video/Makefile	2003-08-01 17:06:43.000000000 +0200
@@ -16,7 +16,7 @@ obj-m		:=
 obj-n		:=
 obj-		:=
 
-SUB_DIRS     := 
+SUB_DIRS     := bttv9
 MOD_SUB_DIRS := $(SUB_DIRS)
 ALL_SUB_DIRS := $(SUB_DIRS)
 
@@ -37,6 +37,7 @@ obj-$(CONFIG_VIDEO_DEV) += videodev.o v4
 obj-$(CONFIG_BUS_I2C) += i2c-old.o
 obj-$(CONFIG_VIDEO_BT848) += bttv.o msp3400.o tvaudio.o \
 	tda7432.o tda9875.o tda9887.o tuner.o
+subdir-$(CONFIG_VIDEO_BT848_9) += bttv9
 obj-$(CONFIG_SOUND_TVMIXER) += tvmixer.o
 
 obj-$(CONFIG_VIDEO_ZR36120) += zoran.o
